### PR TITLE
RegistrationTable Function

### PR DESCRIPTION
**Pull request by jacqueschen1**
- Create registrationtable functions to add/update entries
- Functions to get entries by studentID and by eventID
- Remove database creation from serverless to avoid deployment trying to create new DB tables in the future
- Hopefully fixed serverless deployment bug with cloudwatch logs: https://github.com/serverless/serverless/releases/tag/v1.6.0

Closes: #6

### CODE REVIEW

File: handlers/reg.js [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      TableName: 'biztechRegistration'
+  };
+
+  await docClient.put(params).promise()
+
+  // Update Event count
+  let number = '';
+  let updateExpression = 'set ';
+  switch(data.status) {
+    case 'R':
```
Comment (ianmah) [inline handlers/reg.js]: Is status a query parameter in the request body? Could it just be `regNum` instead of `R`?

File: handlers/reg.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      updateExpression += number + ' \= ' + number + ' \+ :incr,';
+    }
+  }
+  let expressionAttributeValues = {':incr': 1};
+
+  // Update timestamp
+  updateExpression += "updatedAt = :updatedAt";
+  expressionAttributeValues[':updatedAt'] = timestamp;
+
+  console.log(updateExpression);
```
Comment (ianmah) [inline handlers/reg.js]: Maybe we should label this console log

File: handlers/reg.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+      statusCode: 500,
+      body: error
+    };
+    callback(null, response);
+    return;
+  });
+  
+};
+
+// Find entries by studentID
```
Comment (ianmah) [inline handlers/reg.js]: We should add (in comments) what these queries return

File: handlers/reg.js [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+      callback(null, response);
+    })
+    .catch(error => {
+      console.error(error);
+      callback(new Error('Unable to query registration table'));
+      return;
+    });
+}
+
+// Find entries by eventID
```
Comment (ianmah) [inline handlers/reg.js]: Same here

File: serverless.yml [hunk 5]
```diff
@@ -96,34 +97,29 @@ functions:
...
-          AttributeDefinitions:
-            - AttributeName: id
-              AttributeType: S
-          KeySchema:
-            - AttributeName: id
-              KeyType: HASH
-          ProvisionedThroughput:
-            ReadCapacityUnits: 1
-            WriteCapacityUnits: 1
+  regCreate:
```
Comment (ianmah) [inline serverless.yml]: I'd prefer "reg" -> "registration"

File: handlers/reg.js [hunk 6]
```diff
@@ -0,0 +1,147 @@
+'use strict';
+
```
Comment (ianmah) [inline handlers/reg.js]: Any reason for 'reg' instead of 'registration'?

File: handlers/reg.js (hunk 1 above)
Comment (jacqueschen1) [reply handlers/reg.js]: `R`, `C`, etc. are the statuses that are being store in registrationTable, while `regNum`, `checkedNum` etc. is the count being store in events table. If you want we could change both to something more compatible like `registered`, `checkedIn`, `waitlisted`, etc.

File: handlers/reg.js (hunk 6 above)
Comment (jacqueschen1) [reply handlers/reg.js]: No, it was just shorter to write haha

File: handlers/reg.js (hunk 1 above)
Comment (ianmah) [reply handlers/reg.js]: I think it'd be helpful to use words instead of letters, and maybe `registeredCount` or `registeredNum` and `checkedInCount`

**Pull request merged**